### PR TITLE
Added stdout support on js submissions

### PR DESCRIPTION
**Pull request by lypnol**

### CODE REVIEW

**Comment by badouralix** [comment]
https://github.com/doloopwhile/PyExecJS/issues/62 😠

File: submissionJs.py [hunk 1]
```diff
@@ -18,11 +18,37 @@ def language(self):
...
+		answer, stdout = self.compiled.call('console.run', s)
+		if stdout:
+			print(stdout)
+		return answer
 
 class SubmissionJsGenerator:
 	def __init__(self, source):
-		self.compiled = execjs.compile(source)
+		console = """
+var console = {
```
Comment (badouralix) [inline submissionJs.py:29]: Pretty nice hack

Comment (lypnol) [reply submissionJs.py:29]: Let's merge ?

Comment (badouralix) [reply submissionJs.py:29]: Yup 🚢

**Pull request merged**